internal/flag/flag.go

```diff
@@ -529,9 +529,16 @@ func JSONOutput() Bool {
 	}
 }
 
-func ProcessGroup() String {
+func ProcessGroup(desc string) String {
+
+	if desc == "" {
+		desc = "The target process group"
+	}
+
 	return String{
 		Name:        flagnames.ProcessGroup,
```

Comment:
Does changing the constant from "process-group" to "group" break anything?

Reply:
It still has the `process-group` alias, so invocations will work. It will just show up as `group` in help. We had variations on this before. Now all three options work everywhere, and `group` is shown as the option.

Reply:
You'll have to update the occurrences of `flag.GetString`, because it operates on the Name.

e.g.:
https://github.com/superfly/flyctl/blob/6cf80bccc8832537d4f50aa6c19738d53d308c18/internal/command/scale/count.go#L62C10-L62C10

Reply:
Ah right, good call

Reply:
Fixed that